app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -150,7 +150,7 @@ class SearchFragment : Fragment(), UserInteractionHandler {
                 requireContext().getString(R.string.voice_search_content_description),
                 visible = {
                     requireContext().settings().shouldShowVoiceSearch &&
-                        FeatureFlags.voiceSearch && speechIsAvailable()
+                        speechIsAvailable()
```

Comment:
nit - could we move the entire lambda param to be 1 line now? (assuming it fits) 🤤